Classes&Inheritance.js

```diff
@@ -0,0 +1,120 @@
+"use strict";
+
+//General class
+
+class Builder {
+    constructor(startValue = "") {
+        this.value = startValue;
+    }
+
+    get() {
+        return this.value;
+    }
+}
+
+//Child Class - IntBuilder in ES6
+
+class IntBuilder extends Builder {
+    constructor(startValue = 0) {
+        super(startValue);
+    }
+
+    static random(from, to) {
+        return Math.floor(Math.random() * (Math.floor(to) - Math.ceil(from) - 1) + Math.ceil(from));
+    }
+
+    plus(...numbers) {
+        for (let number of numbers) {
+            this.value += number;
+        }
+        return this;
+    }
+
+    minus(...numbers) {
+        for (let number of numbers) {
+            this.value -= number;
+        }
+        return this;
+    }
+
+    multiply(number) {
+        this.value *= number;
+        return this;
+    }
+
+    divide(number) {
+        this.value /= number;
+        return this;
+    }
+
+    mod(number) {
+        this.value %= number;
+        return this;
+    }
+}
+
+// Child Class - StrBuilder in ES5
+
+function StringBuilder(startValue = '') {
+    const instance = Reflect.construct(Builder, [startValue], StringBuilder);
+    Object.assign(this, instance);
+}
+
+StringBuilder.prototype = Object.create(Builder.prototype);
+StringBuilder.prototype.constructor = StringBuilder;
+
+StringBuilder.prototype.plus = function (...strings) {
+    for (let string of strings) {
+        this.value += string;
+    }
+    return this;
+}
+
+StringBuilder.prototype.minus = function (number) {
+    this.value = this.value.slice(0, (this.value.length - number));
+    return this;
+}
+
+StringBuilder.prototype.multiply = function (number) {
+    this.value = this.value.repeat(number);
+    return this;
+}
+
+StringBuilder.prototype.divide = function (number) {
+    this.value = this.value.slice(0, (Math.floor(this.value.length / number)));
```

Comment:
```suggestion
    this.value = this.value.slice(0, number);
```
I think, it has to work

Reply:
Done.